pkg/pillar/types/domainmgrtypes.go

```diff
@@ -43,6 +43,11 @@ type DomainConfig struct {
 
 	// MetaDataType for select type of metadata service for app
 	MetaDataType MetaDataType
+
+	// Privileged indicates that we want to run app as a service
+	Privileged bool
```

Comment:
Isn't this name a bit misleading? IsSystemService or IsServiceContainer seems more clear to me.
For instance the edgeview container does not have all priviledges.